.github/workflows/coverage-linux.yml

```diff
@@ -18,7 +17,6 @@ on:
       - 'benchmark/**'
       - 'deps/**'
       - 'doc/**'
-      - 'tools/**'
```

Comment:
Same here as prior suggestion to use `paths:` instead of `paths-ignore` and use `!` to negate each path, and then explicitly add `tools/inspector_protocol/**`.